client/cmdhfmf.c

```diff
@@ -574,8 +612,9 @@ int CmdHF14AMfDump(const char *Cmd) {
 		return 0;
 	}
 	
-	if ((fin = fopen("dumpkeys.bin","rb")) == NULL) {
-		PrintAndLog("Could not find file dumpkeys.bin");
+	fptr = GenerateFilename("hf-mf-","-key.bin");

```

Comment:
So, question,
how is the file going to be loaded without the card on the antenna?

Reply:
Never think of the offline attack. Will take a look if the new code will break things tonight

Reply:
That should be fine since you can't dump a tag without a tag on the antenna. If you must place a tag then the filename can be generated. Btw, I added option to allow user to specify the key file to read and data file to save.